DiscordBot/Commands/EventCommands.cs

```diff
@@ -74,9 +74,15 @@ public async Task CreateEvent(CommandContext context, InteractivityModule intera
                 return;
             }
 
-            if (!DateTimeOffset.TryParse(eventTimeResponse, out var eventTime))
+            DateTimeOffset eventTime;
```

Comment:
It might be a bit neater to have a separate method to `GetUserTimeReponse` which will get the response, try to parse it, write the error to the user and return null if the parsing fails. Then here you'll just need the null check like with the other user inputs. What do you think? 🙂

Reply:
Sounds like a good idea, it should also many any future time queries easier.